src/main/java/cubicchunks/asm/mixin/core/common/MixinWorldServer.java

```diff
@@ -104,7 +106,11 @@
 
 		for (int cubeX = spawnCubeX - spawnDistance; cubeX <= spawnCubeX + spawnDistance; cubeX++) {
 			for (int cubeZ = spawnCubeZ - spawnDistance; cubeZ <= spawnCubeZ + spawnDistance; cubeZ++) {
+				// Preload column
+				// serverCubeCache.loadColumn(cubeX, cubeZ, LOAD_OR_GENERATE); TODO reimplement server cube cache
 				for (int cubeY = spawnCubeY + spawnDistance; cubeY >= spawnCubeY - spawnDistance; cubeY--) {
+					// TODO reimplement server cube cache
+					// serverCubeCache.asyncLoadCube(cubeX, cubeY, cubeZ, col -> {});
```

Comment:
Yes, sorry, that was from before I decided to split it up. but we can leave it sync, no problem